app/helpers/questions_helper.rb

```diff
@@ -56,4 +56,12 @@ def last_question_seen?
   def all_questions
     I18n.t("coronavirus_form.groups").map { |_, group| group[:questions].keys if group[:title] }.compact.flatten.map(&:to_s)
   end
+
+  def remove_questions(questions)
+    updated_questions_to_ask = questions_to_ask
+    questions.each do |question|
+      updated_questions_to_ask.delete(question)
+    end
+    updated_questions_to_ask
```

Comment:
Are `questions` and `questions_to_ask` both arrays? If so, you could do `questions_to_ask - questions` here.

Reply:
Nice one!  I didn't spot this.